src/styles/Contact.css

```diff
@@ -81,7 +85,7 @@
   right: 0;
   margin-left: auto;
   margin-right: auto;
-  font-size: 2vw;
+  font-size: 1vw;
   color: white;
   vertical-align: middle;
```

Comment:
Contact.css: `vertical-align: middle;` can be moved from `#Header`, `#Chat`, `#Help`, `#Follow`, `#Copyright`  to `.Contact` and declared once just like in About.css.